manuals/2.0/ja/session.md

```diff
@@ -1,23 +1,22 @@
 ---
-layout: docs2-en
-title: Session
-permalink: /manuals/2.0/en/session/
+layout: docs2-ja
+title: セッション
```

Comment:
yey. I like this. I was wondering whether it was good to be in English for a Japanese translation , ie why didn't mentioned on other commit . Good work. :+1:  .